src/forms/NewStudyForm.js

```diff
@@ -1,351 +1,453 @@
+              {workflowType.length > 0 && (
+                <List.Item>
+                  Cavatica harmonization projects
+                  <List.List>
+                    {workflowType.map(type => (
+                      <List.Item key={type}>
+                        {
+                          workflowOptions.filter(obj => obj.value === type)[0]
+                            .text
+                        }
                       </List.Item>
-                    )}
-                  </List>
-                </Container>
-              }
-              confirmButton={
-                <Button
-                  primary
-                  floated="right"
-                  type="submit"
-                  loading={isSubmitting}
-                >
-                  SUBMIT
-                </Button>
-              }
+                    ))}
+                  </List.List>
+                </List.Item>
+              )}
+            </List>
+          </Container>
+        }
+        confirmButton={
+          <Button primary floated="right" type="submit" loading={isSubmitting}>
+            SUBMIT
+          </Button>
+        }
+      />
+    </>
+  );
+};
+
+const NewStudyForm = ({submitValue, apiErrors}) => {
+  const [confirmOpen, setConfirmOpen] = useState(false);
+  const [workflowType, setSelection] = useState([
+    'bwa_mem',
+    'gatk_haplotypecaller',
+  ]);
+  const [focused, setFocused] = useState('');
+  const [activeStep, setActiveStep] = useState('info');
+  return (
+    <Formik
+      initialValues={{
+        externalId: '',
+        name: '',
+        shortName: '',
+        description: '',
+        releaseDate: '',
+        version: '',
+        attribution: '',
+        anticipatedSamples: 0,
+        awardeeOrganization: '',
+      }}
+      validate={values => {
+        let errors = {};
+        if (!values.externalId) {
+          errors.externalId = 'Required';
+        }
+        if (!values.name) {
+          errors.name = 'Required';
+        }
+        if (values.anticipatedSamples < 0) {
+          errors.anticipatedSamples = 'Invalid number of anticipated samples';
+        }
+        if (values.version.length > 10) {
+          errors.version = 'Maximum 10 characters long';
+        }
+        return errors;
+      }}
+      onSubmit={(values, {setSubmitting}) => {
+        setSubmitting(false);
+        setConfirmOpen(false);
+        var inputObject = values;
+        if (values.releaseDate.length === 0) {
+          inputObject.releaseDate = null;
+        }
+        submitValue({input: inputObject, workflowType: workflowType});
+      }}
+    >
+      {formikProps => (
+        <Fragment>
+          {apiErrors && (
+            <Message
+              negative
+              icon="attention"
+              header="Error"
+              content={apiErrors}
+            />
+          )}
+          <Step.Group attached="top" fluid widths={3}>
+            <Step
```

Comment:
this could be more compact by pushing each of these into an object keyed by their step number
```
STUDY_STEPS = {
0: {
  name: '',
  title: '',
  desc:'',
  icon: '',
  comp: StepComponent
},
 }
```

then you can just map over them here 

```
Object.values(STUDY_STEPS).map(stepObj => <Step  />) 
```